service-impl/resource-service-impl/hadoop-resource-service/pom.xml

```diff
@@ -21,6 +21,58 @@
             <artifactId>shade-hadoop-client</artifactId>
             <version>${project.parent.version}</version>
         </dependency>
+        <!-- This only needs to be here for AWS access-->
```

Comment:
These dependancy additions are not required as they are already in the shaded-hadoop-client dependancy